src/components/Profile/ProfileInfo/ProfileStatusWithHooks.js

```diff
@@ -0,0 +1,41 @@
+import React, { useState, useEffect } from 'react';
+
+const ProfileStatusWithHooks = (props) => {
+  const [editMode, setEitMode] = useState(false);
+  const [status, setStatus] = useState(props.status);
+
+  useEffect(() => {
+    setStatus(props.status);
+  }, [props.status]);
+
+  const activateEditMode = () => {
+    setEitMode(true);
+  };
+
+  const deactivateEditMode = () => {
+    setEitMode(false);
+    props.updateStatus(status);
+  };
+
+  const onStatusChange = (e) => {
+    setStatus(e.currentTarget.value);
+  };
+
+  return (
+    <div>
+      {!editMode
```

Comment:
Consider starting with "editMode" true expression.
`editMode ? ... : ...`

Reply:
i do it